mixer/adapter/solarwinds/appoptics/mock_client.go

```diff
@@ -0,0 +1,19 @@
+// Copyright 2017 Istio Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package appoptics
+
+type MockClient struct {
```

Comment:
is this full type needed here?  and should this file be `mock_client_test.go` to avoid usage in non-test code?

Reply:
sorry, this was a left over from transitioned code. I have removed it now.